rotkehlchen/db/filtering.py

```diff
@@ -333,6 +342,17 @@ def prepare(self) -> Tuple[List[str], List[Any]]:
         return [f'{self.asset_key}=?'], [self.asset.identifier]
 
 
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBIgnoreValuesFilter(DBFilter):
+    column: str
+    values: List[Any]
+
+    def prepare(self) -> Tuple[List[str], List[Any]]:
+        if len(self.values) == 0:
+            return [], []
+        return [f'{self.column} NOT IN ({", ".join(["?"] * len(self.values))})'], [self.values]
```

Comment:
This is tricky. Probably for now we can merge as is but this is incomplete and can hide a bug.

1. Don't use `Any` as type here. Use as exact types as you can and we know that this is only used for strings here so use List[str].
2.  If the list ever ends up containing a `None` this will fail very sneakly. https://stackoverflow.com/questions/129077/null-values-inside-not-in-clause
Null checks don't work with IN. If we ever need a null check here (say the list can have `None`) then it should be added as an addition OR self.column IS NULL or IS NOT NULL depending on query

Reply:
oh okey. I was using str in the previous commit but wanted to generalize. From now on I'll go restrictive and expand types as we need it